frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -25,13 +25,11 @@ const viewAPIExplorerMenuItem = (name: string, apiExplorerLink: string) => ({
   href: apiExplorerLink,
 });
 
-const getOauthMenuItems = () => (
-  Object.keys(addIDPItems).map(oauthItem => {
-    return {
-      label: `Add ${addIDPItems[oauthItem]} IDP`,
-      href: `/settings/idp/${oauthItem}`,
-    };
-  })
+const getOAuthMenuItems = () => (
```

Comment:
This doesn't really need to be a function. It could be a const var since it won't change

Reply:
@spadgett sounds good.